components/TextInput.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { useState } from "react";
+
+const TextInput = (props) => {
+  const [value, setValue] = useState("");
+
+  const handleChange = (event) => {
```

Comment:
@pasternakewa You have created input with internal state, in form You have multiple inputs what should modify Your state. Please take a look at:
https://pl.reactjs.org/docs/forms.html